src/components/playerOverview/playerDetails/PlayerDetailsInfoCard.tsx

```diff
@@ -23,6 +28,19 @@ export const PlayerDetailsInfoCard = React.memo(function PlayerDetailsInfoCard({
     const profileId = potentialProfileId ?? "";
     const profile = useSelector((state: AppState) => ProfileSelectors.getProfile(state, profileId));
 
+    const moxfieldProfile: MoxfieldProfile | undefined = useSelector((state: AppState) => {
+        if (state.profiles.moxfieldProfiles !== undefined && profile?.moxfieldId !== undefined) {
```

Comment:
your next task is to make this into a real selector. we can table this for now though